Search/Response.swift

```diff
@@ -8,16 +8,17 @@
 
 import Foundation
 
-public class Response: NSObject {
+public final class Response: NSObject {
 
     // MARK: Declaration for string constants to be used to decode and also serialize.
-    internal let kResponseResultsKey: String = "results"
-    internal let kResponseHeaderKey: String = "header"
+    internal static let ResultsKey: String = "results"
+    internal static let HeaderKey: String = "header"
+    internal static let DpaKey: String = "DPA"
 
-    public let results: [Results]
+    public let results: [Dpa]
```

Comment:
Hmm, what's a `Dpa` would be my thought as a user. Could we not have a single standard `SearchResult` class that understands the format regardless of datasource? Or if we're only using DPA, I think something more generic as a name would still be preferable?